cli/pkg/kctrl/cmd/package/repository/release/release.go

```diff
@@ -140,8 +140,8 @@ func (o *ReleaseOptions) Run() error {
 		if err != nil {
 			return err
 		}
+		defer os.RemoveAll(filepath.Join(wd, LockOutputFolder))
```

Comment:
```suggestion
		defer os.RemoveAll(tmpImgpkgFolder)
```

Reply:
Removed usage of wd as it is not necessary and started using `LockOutputFolder` directly